tests/tests/test-precompile/test-precompile-proxy.ts

```diff
@@ -463,3 +464,97 @@ describeDevMoonbeam("Pallet proxy - shouldn't accept instant for delayed proxy",
     expect(afterCharlethBalance - beforeCharlethBalance).to.be.eq(0n);
   });
 });
+
+describeDevMoonbeam("Pallet proxy - should transfer using value", (context) => {
+  it("should transfer using value", async () => {
+    const {
+      result: { events },
+    } = await context.createBlock(
+      createTransaction(context, {
+        ...ALITH_TRANSACTION_TEMPLATE,
+        to: PRECOMPILE_PROXY_ADDRESS,
+        data: PROXY_INTERFACE.encodeFunctionData("addProxy", [
+          BALTATHAR_ADDRESS,
+          CONTRACT_PROXY_TYPE_ANY,
+          0,
+        ]),
+      })
+    );
+    expectEVMResult(events, "Succeed");
+
+    const beforeAlithBalance = BigInt(await context.web3.eth.getBalance(ALITH_ADDRESS));
+    const beforeCharlethBalance = BigInt(await context.web3.eth.getBalance(CHARLETH_ADDRESS));
+    const value = BigInt(context.web3.utils.toWei("10", "ether"));
+
+    const {
+      result: { events: events2 },
+    } = await context.createBlock(
+      createTransaction(context, {
+        ...BALTATHAR_TRANSACTION_TEMPLATE,
+        to: PRECOMPILE_PROXY_ADDRESS,
+        data: PROXY_INTERFACE.encodeFunctionData("proxy", [ALITH_ADDRESS, CHARLETH_ADDRESS, []]),
+        value: value.toString(),
+      })
+    );
+
+    expectEVMResult(events2, "Succeed");
+
+    const afterAlithBalance = BigInt(await context.web3.eth.getBalance(ALITH_ADDRESS));
+    const afterCharlethBalance = BigInt(await context.web3.eth.getBalance(CHARLETH_ADDRESS));
+    const afterProxyPrecompileBalance = BigInt(
+      await context.web3.eth.getBalance(PRECOMPILE_PROXY_ADDRESS)
+    );
+
+    expect(afterAlithBalance - beforeAlithBalance).to.be.eq(-value);
```

Comment:
I think it's a bit confusing to compare against a negative value. Instead, you could do:
```suggestion
    expect(beforeAlithBalance - afterAlithBalance).to.be.eq(value);
```